transport/ssh.go

```diff
@@ -64,9 +65,23 @@ func sshConnect(host, port, user string, strictHostKeyChecking bool) (*ssh.Clien
 			}
 		}
 
+		var authmethod []ssh.AuthMethod
+		if key, err := ioutil.ReadFile(identityfile); err != nil {
+			return nil, fmt.Errorf("Unable to read private key, falling back to SSH agent: %v", err)
+			// activity.Record(ctx, "Unable to read private key, falling back to SSH agent: %v", err)
```

Comment:
The missing piece is that you need the `context.Context` (often given the variable name `ctx`) object passed to whatever method calls `sshConnect`.

Reply:
... should be able to do this one fast too. Thank you.

Reply:
Ready for review